src/framework/midi/internal/synthesizersregister.cpp

```diff
@@ -23,6 +23,7 @@ using namespace mu::midi;
 void SynthesizersRegister::registerSynthesizer(const SynthName& name, std::shared_ptr<ISynthesizer> s)
 {
     m_synths[name] = s;
+    audioEngine()->startSynthesizer(s);
```

Comment:
Synthesizers Register is just register, storage, there is shouldn't something like this.
The second problem, this is the method called from registerExports, at the moment some interfaces maybe registred, some not yet, t.e audio engine maybe just not be registered at the moment, so we shouldn't do anything at the moment,  we should do like this in an onInit.